app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -18,23 +18,30 @@ import {
 
 @Component({
   selector: 'syndesis-import-integration-component',
-  templateUrl: './integration-import.component.html'
+  templateUrl: './integration-import.component.html',
+  styleUrls: ['./integration-import.component.scss']
 })
 export class IntegrationImportComponent implements OnInit {
   public uploader: FileUploader;
+  public hasBaseDropZoneOver = false;
```

Comment:
Usually, when initializing boolean class members as `false`, we would just annotate the `Boolean` token, since it makes code more expressive, given the fact that boolean values are initialized as `null`, which equals `false` (unless we coerce type comparison later on, which is seldom the case).